parachains/runtimes/starters/shell/src/lib.rs

```diff
@@ -14,6 +14,11 @@
 // You should have received a copy of the GNU General Public License
 // along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
 
+//! # Shell Runtime
+//!
+//! The Shell runtime defines a minimal parachain. It can listen for a downward message authorizing
+//! an upgrade into another parachain.
```

Comment:
Would be great for someone to say why this is a handy technique sometimes. If it's often not appropriate to do this then we should recommend that in general they should not need to do this and can start with their own full runtime.